lib/transport-common.js

```diff
@@ -71,30 +73,21 @@ const newConnectFn = (
 const newConnectAndInspectFn = (connFactory, transportClass) => {
   const connect = newConnectFn(connFactory, transportClass);
   return (appName, client, interfaces, ...options) => {
-    const callback = common.extractCallback(options);
+    const cb = common.cbExtract(options);
     connect(appName, client, ...options, (error, connection) => {
       if (error) {
-        callback(error);
+        cb(error);
         return;
       }
-      const proxies = Object.create(null);
-      let errored = false;
-      const len = interfaces.length;
-      let count = 0;
+      const pCollector = metasync.collect(interfaces.length);
```

Comment:
Maybe something like `proxiesCollector` or even just `collector`. That lone "p" letter doesn't help readability much and looks like hungarian notation.